samples/msal-node-samples/b2c-auth-code-pkce/index.js

```diff
@@ -16,10 +17,10 @@ const SERVER_PORT = process.env.PORT || 3000;
  */
 const publicClientConfig = {
     auth: {
-        clientId: "67ffe8a0-db42-464a-88d7-c26cbdd06ce2", 
+        clientId: '67ffe8a0-db42-464a-88d7-c26cbdd06ce2',
```

Comment:
Are we choosing to leave in the client id here intentionally?

Reply:
This one, yes! It's a demo b2c tenant I've been using for samples as a convenience, since B2C tenant setup is a bit lengthy process